dev/drivers/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.sh

```diff
@@ -24,7 +24,7 @@ export job=${PBS_JOBNAME:-jevs_mesoscale_grid2obs_plots}
 export jobid=$job.${PBS_JOBID:-$$}
 export SITE=$(cat /etc/cluster_name)
 export USE_CFP=YES
-export nproc=128
+export nproc=64
 export evs_run_mode="production"
 
```

Comment:
@malloryprow I made this change in my current PR #603 .

Reply:
This PR isn't going into develop